sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -102,6 +103,11 @@
     private static final CrashesListener DEFAULT_ERROR_REPORTING_LISTENER = new DefaultCrashesListener();
 
     /**
+     * Timeout for getting crash for the last session.
+     */
+    private static final int GET_LAST_SESSION_CRASH_REPORT_TIMEOUT = 10000;
```

Comment:
It sounds like a PM question for requirement.
-> Got the same opinion. Removing timeout and just wait forever.